### DISCUSSION TITLE
[FR] - State Timeline- ability to remove X-Axis (ie the time line text)

### DISCUSSION THREAD

**Discussion by bob454522** [grafana/grafana#71005] [Dashboards, Panels & Visualizations]
Grafana v 10.0.1 - 

Feature request-  Need ability to remove the time-line text (ie the X-Axis) from the bottom of a State Timeline panel type.

I have seen others ask this, and get official replies from grafana staff stateing there is no way currently to hide the timeline at the bottom of a State Timeline panel type. 
with space at a premium on many dashboards, and the fact that the state timeline produces beautiful detailed data, it would be great if there was an option to hide the time frame at the bottom.

IE hide what i have indicated in image below.  

Thanks, LOVE GRAFANA!

additionally-  (there is no way to hide the series name either, with the best work around being to give the series name a single character alias:  such as - or .   it will still get displayed but will not take up much space).

![image](https://github.com/grafana/grafana/assets/60830628/2cf0c426-ad30-4875-88f8-f4458975bc28)

**Comment by leeoniya** [comment]
we've seen requests to also hide the y axis: https://github.com/grafana/grafana/discussions/38847

maybe we should expose both 🤔

**Comment by fzyzcjy** [comment]
Hi, is there any updates? Thanks!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82614. Feel free to continue the discussion around this there. Thank you!
